dataloader.go

```diff
@@ -20,18 +20,47 @@ import (
 // different access permissions and consider creating a new instance per
 // web request.
 type Interface interface {
-	Load(context.Context, interface{}) Thunk
-	LoadMany(context.Context, []interface{}) ThunkMany
-	Clear(context.Context, string) Interface
+	Load(context.Context, Keyer) Thunk
+	LoadMany(context.Context, KeyList) ThunkMany
+	Clear(context.Context, Keyer) Interface
 	ClearAll() Interface
-	Prime(ctx context.Context, key interface{}, value interface{}) Interface
+	Prime(ctx context.Context, key Keyer, value interface{}) Interface
+}
+
+// StringKey implements the Key interface for a string
+type StringKey string
```

Comment:
What do you think about including a `StringKey`-type helper for creating a `KeyList`?